activiti-core-common/activiti-expression-language/src/main/java/org/activiti/core/el/ExpressionResolver.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2010-2020 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.activiti.core.el;
+
+import java.util.Map;
+
+public interface ExpressionResolver {
```

Comment:
This won't be used by Activiti Engine, since it has its own ExpressionManager. It's intended for activiti-cloud connectors and any other Activiti extensions usage.